### ISSUE TITLE
Error with logging

### ISSUE THREAD

**Issue by petersw730** [gjr80/weewx-bloomsky#5]
I'm running WeeWx on a Pi4, but facing some issues with Bloomsky module.  Everything seems to be set correctly, but I get the following error:

Sep  5 16:32:59 pi4 systemd[1]: Starting LSB: weewx weather system...
Sep  5 16:33:00 pi4 weewx[31392] INFO __main__: Initializing weewx version 4.5.1
Sep  5 16:33:00 pi4 weewx[31392] INFO __main__: Using Python 3.7.3 (default, Jan 22 2021, 20:04:44) #012[GCC 8.3.0]
Sep  5 16:33:00 pi4 weewx[31392] INFO __main__: Platform Linux-5.10.60-v7l+-armv7l-with-debian-10.10
Sep  5 16:33:00 pi4 weewx[31392] INFO __main__: Locale is 'en_US.UTF-8'
Sep  5 16:33:00 pi4 weewx[31392] INFO __main__: PID file is /var/run/weewx.pid
Sep  5 16:33:00 pi4 weewx[31400] INFO __main__: Using configuration file /etc/weewx/weewx.conf
Sep  5 16:33:00 pi4 weewx[31400] INFO __main__: Debug is 0
Sep  5 16:33:00 pi4 weewx[31400] INFO weewx.engine: Loading station type Bloomsky (user.bloomsky)
Sep  5 16:33:00 pi4 weewx[31400] CRITICAL __main__: Caught unrecoverable exception:
Sep  5 16:33:00 pi4 weewx[31400] CRITICAL __main__:     ****  name 'logging' is not defined

The Traceback shows:

 ****  Traceback (most recent call last):
****    File "/usr/share/weewx/weewxd", line 151, in main
****      engine = weewx.engine.StdEngine(config_dict)
****    File "/usr/share/weewx/weewx/engine.py", line 81, in __init__
****      self.setupStation(config_dict)
****    File "/usr/share/weewx/weewx/engine.py", line 108, in setupStation
****      __import__(driver)
****    File "/usr/share/weewx/user/bloomsky.py", line 185, in <module>
****      log = logging.getLogger(__name__)
****  NameError: name 'logging' is not defined

Any suggestions?

**Comment by gjr80** [comment]
Yep, looks like I left an old `logging` call in the code. Try downloading the [v2.0.2 release](https://github.com/gjr80/weewx-bloomsky/releases/tag/v2.0.2), just download the extension package and install over the top of your current version. No need to uninstall. I have not tested v2.0.2 but it was a simple and obvious error so should be fine. Please let me know how that goes.
